src/store/index.jsx

```diff
@@ -0,0 +1,51 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+export const initialState = {
+  user: {
+    isAuth: false, // default value - false. After success login - true
+    name: '', // default value - empty string. After success login - name of user
+    email: '', // default value - empty string. After success login - email of user
+    token: '', // default value - empty string or token value from localStorage. After success login - value from API /login response.
+  },
+  // todo: initialize this courses to an empty array, delete mock data
+  courses: [], // list of courses. Default value - empty array. After success getting courses - value from API /courses/all response.
```

Comment:
Please, use different reducer for your courses state regarding task requirements